config/jobs/kubernetes/sig-node/README.md

```diff
@@ -0,0 +1,190 @@
+## Signode CI Test Coverage Guidance
+
+
+## Variables
+
+
+<table>
+  <tr>
+   <td><strong>Variables</strong>
+   </td>
+   <td><strong>Values</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>cri
+   </td>
+   <td>Containerd, CRI-O
+   </td>
+  </tr>
+  <tr>
+   <td>cgroup
+   </td>
+   <td>cgroupv1, cgroupv2
+   </td>
+  </tr>
+  <tr>
+   <td>os
+   </td>
+   <td>cos, ubuntu, fedora
+   </td>
+  </tr>
+  <tr>
+   <td>release branch
+   </td>
+   <td>N-3, N-2, N-1, N, main
+   </td>
+  </tr>
+</table>
+
+
+
+## Naming Convention
+
+{cri}-{cgroup}-{os}-{test-type}-release-{release-branch}
+
+**Note:** Specify variable value in the name only if it isn’t set to default in a test tab.
+
+
+## Test Tabs
+
+#### 1. **sig-node-default (i.e. rename sig-node-release-blocking)**
+
+<table>
```

Comment:
We'll need a dashboard for:

- for each architecture
- for each cloud

Reply:
I will discuss this in tomorrow's sig CI meeting and accommodate this